web/client/components/catalog/Catalog.jsx

```diff
@@ -360,7 +362,7 @@ class Catalog extends React.Component {
                             </Form>)}
                     footer={this.renderPagination()}>
                             <div>
-                                {this.renderResult()}
+                                { this.props.loading ? <div className="catalog-results empty"><Spinner spinnerName="circle" noFadeIn overrideSpinnerClassName="spinner"/></div> : this.renderResult() }
```

Comment:
Instead of Spinner we should use Loader from /misc folder to keep the style aligned with other loader.
I suggest to use size 176px similar to the main spinner of Loading MapStore.
Probably it's better to use a different class for the container that describe the loading action:
proposal -> `catalog-results-loading` instead of  `catalog-results empty`

Reply:
@allyoucanmap what about removing the spinner inside the search button?

Reply:
I think @allyoucanmap has forgotten to remove it in the snapshot but we agreed it before that I should remove all other spinners in favour of a single visible large spinner